contracts/test/nativeTransfer.mjs

```diff
@@ -0,0 +1,182 @@
+import hardhat from "hardhat"
+import chai from "chai"
+import {getGasPrice, getTxReceipt} from "../utils/tx.js"
+
+import {isGwMainnetV1} from "../utils/network.js"
+
+const {ethers} = hardhat
+const {expect} = chai
+const {BigNumber} = ethers
+
+let gasPrice, EOA0, EOA1, newEOA0, CA0
+
+if (!isGwMainnetV1()) {
+    gasPrice = await getGasPrice(ethers.provider)
+    EOA0 = (await ethers.getSigners())[0].address
+    EOA1 = (await ethers.getSigners())[1].address
+    newEOA0 = ethers.Wallet.createRandom().address
+    let baseFallbackReceive = await ethers.getContractFactory("baseFallbackReceive")
+    let contract = await baseFallbackReceive.deploy()
+    await contract.deployed()
+    CA0 = contract.address
+}
```

Comment:
If you have things to do before testing, maybe you can try the following solution instead of using `.mjs`? 
Or is there any other reason to use `.mjs`?

Here's an example on how to do things before testing:
```ts
describe("native transfer", () => {
  before("before testing", async () => {
    // do things before testing
  });
  describe("transfer success", () => {...});
  describe("transfer failed", () => {...});
});
```

Reply:
> If you have things to do before testing, maybe you can try the following solution instead of using `.mjs`? Or is there any other reason to use `.mjs`?
> 
> Here's an example on how to do things before testing:
> 
> ```ts
> describe("native transfer", () => {
>   before("before testing", async () => {
>     // do things before testing
>   });
>   describe("transfer success", () => {...});
>   describe("transfer failed", () => {...});
> });
> ```

The mocha test framework does not support data-driven, so the initialized test data must be passed in outside the `it` code block through the for loop. The  `async ()`  in `before` can only take effect in the `it` code block. If it is placed outside `it`, it will be `undefined`, so it is best to use `.mjs` to support top-level await to initialize test data.
```ts
let EOA0

before(async () => {
    EOA0 = (await ethers.getSigners())[0].address
})

describe("use EOA0 before it", function () {
    console.log(EOA0)
    it("it code block", async () => {
    })
})

//print out undefine
```